core/src/config.rs

```diff
@@ -139,8 +139,8 @@ mod tests {
 
     use super::*;
 
-    const CONFIGURATION_PATH: &str = "tests/test_config.json";
-    const TRUSTED_PEERS_PATH: &str = "tests/test_trusted_peers.json";
+    const CONFIGURATION_PATH: &str = "../test_configs/core/tests/test_config.json";
+    const TRUSTED_PEERS_PATH: &str = "../test_configs/core/tests/test_trusted_peers.json";
```

Comment:
What is the motivation for test_configs directory outside the crate?
It refers to the structure of external directories from inside the crate.
Would not it cause confusion in the future?
Or the benefits outweigh it?

Reply:
And if do so, filenames under the specific directory can strip `test_` prefix

Reply:
> What is the motivation for `test_configs` directory.

Issue #1237. Essentially we don't want people digging through the crates to find the example configurations. While we could just symlink to them from within the crate on build/test, since we are unlikely to use these crates outside of `iroha`, hard-coding the relative path works just as well.

Reply:
Oh, I see.
Then why only for tests?

Reply:
And I think we can have git watch `.sample.json`s and ignore `.json`s